ui/address/utils/useAddressQuery.ts

```diff
@@ -92,7 +98,7 @@ export default function useAddressQuery({ hash, isEnabled = true }: Params): Add
       };
     },
     placeholderData: [ GET_BALANCE ],
-    enabled: apiQuery.isError || apiQuery.errorUpdateCount > 0,
+    enabled: (apiQuery.isError || apiQuery.errorUpdateCount > 0) && !NO_RPC_FALLBACK_ERROR_CODES.includes(apiQuery.error?.status ?? 999),
```

Comment:
What does `999` stand for? Shouldn't we check if `apiQuery.error?.status` exists before searching for it in the `NO_RPC_FALLBACK_ERROR_CODES` array?